### DISCUSSION TITLE
limited to only one data source for the moment?

### DISCUSSION THREAD

**Discussion by zsdili** [nocodb/nocodb#6981] [Q&A]
How can I connect more data source on my seft-host?

**Comment by zsdili** [comment]
![image](https://github.com/nocodb/nocodb/assets/112945270/bfea4e38-c7b8-41ad-b08b-6fbc01421bf7)

**Comment by dstala** [comment]
@zsdili It is one data source per base. To connect to another data source, you can create another base.

**Reply by zsdili** [reply]
![image](https://github.com/nocodb/nocodb/assets/112945270/02125518-ebd2-4abf-b6b9-68ab01aa4c5b)
Thank you for your reply, but it's a shame to hear that you can only add one database. Isn't it possible to add a lot of different or even different types of databases?
